src/Microsoft.IIS.WebManager/app/header/header.component.ts

```diff
@@ -143,13 +143,17 @@ export class HeaderComponent implements OnDestroy {
             else {
                 if (this._timeout) {
                     clearTimeout(this._timeout);
-                    this._timeout = 0;
+                    this._timeout = null;
                     setTimeout(_ => this._inProgress = false, 300);
                 }
             }
         }));
     }
 
+    private get options() {
```

Comment:
Why do we need this? It's private, the field is private. Why not just access the field?

Reply:
same as above